lambdas/issuecredential/src/main/java/uk/gov/di/ipv/cri/kbv/api/service/EvidenceFactory.java

```diff
@@ -84,102 +82,76 @@ public Object[] create(KBVItem kbvItem, EvidenceRequest evidenceRequest)
         return new Map[] {objectMapper.convertValue(evidence, Map.class)};
     }
 
-    private CheckDetail[] createCheckDetailsWithKbvQuality(KBVItem kbvItem)
-            throws JsonProcessingException {
-        var questionState = objectMapper.readValue(kbvItem.getQuestionState(), QuestionState.class);
+    private CheckDetail[] withKbvQuality(KBVItem kbvItem) throws JsonProcessingException {
+        QuestionState questionState =
+                objectMapper.readValue(kbvItem.getQuestionState(), QuestionState.class);
+        KbvQuestionAnswerSummary answerSummary = kbvItem.getQuestionAnswerResultSummary();
 
-        if (hasPassedWithOneIncorrectAnswer(kbvItem)) {
+        if (batchNumberOfIncorrectAnswer(kbvItem, questionState) == THREE) {
```

Comment:
I'm not 100% sold that we need to use a constant over just the number 3 here but keen to hear what others think

Reply:
Yeah am really indifferent about it. I can remove it